src/OMODFrameworkWrapper.cpp

```diff
@@ -412,3 +416,16 @@ void OMODFrameworkWrapper::displayReadmeSlot(const QString& modName, const QStri
     readmePopup->setAttribute(Qt::WA_DeleteOnClose);
   }
 }
+
+void OMODFrameworkWrapper::showWaitDialogSlot(QString message) {
+  mWaitDialog = new QProgressDialog(message, tr("Cancel"), 0, 0, mParentWidget);
```

Comment:
> The cancelButtonText is the text to display on the cancel button. If QString() is passed then no cancel button is shown.

That'd save you:
* Having a translatable string that doesn't get seen.
* Manually calling `setCancelButton`.
* Having a string localised here that Qt knows how to translate on its own.

Reply:
Done.